front/components/shared/tools_picker/ActionCard.tsx

```diff
@@ -0,0 +1,82 @@
+import { Card, CardActionButton, XMarkIcon } from "@dust-tt/sparkle";
+
+import type { AgentBuilderAction } from "@app/components/agent_builder/types";
+import { isDefaultActionName } from "@app/components/agent_builder/types";
```

Comment:
I should also move and rename these, but I think this PR is getting pretty big so I'd rather do it in a follow up one